LoopKitUI/Views/Information Screens/DeliveryLimitsInformationView.swift

```diff
@@ -59,8 +59,11 @@ public struct DeliveryLimitsInformationView: View {
         VStack(alignment: .leading, spacing: 20) {
             Text(DeliveryLimits.Setting.maximumBolus.title)
             .font(.headline)
-            Text(String(format: LocalizedString("Maximum Bolus is the highest bolus amount that you will allow %1$@ to recommend at one time to cover carbs or bring down high glucose.", comment: "Information about maximum bolus (1: app name)"), appName))
-            .foregroundColor(.secondary)
+            VStack(alignment: .leading, spacing: 20) {
+                    Text(LocalizedString("Maximum Bolus is the highest bolus amount \(appName) will recommend, or allow you to deliver, at one time to cover carbs or bring down high glucose.", comment: "Information about maximum bolus"))
+                    Text(LocalizedString("Your maximum bolus will also set a cap on how much insulin will be allowed for automatic dosing.  \(appName) will restrict automatic dosing to amounts that will not cause your insulin on board to exceed twice your maximum bolus.", comment: "Information about maximum automated insulin on board"))
```

Comment:
"will not cause your insulin on board to exceed twice your maximum bolus" -> "will keep your total active insulin below twice your maximum bolus"

This avoids stating it in the negative, and uses "active insulin" instead of "insulin on board".

Reply:
I think we need to make it clear that manual bolus is not limited by 2 * maxBolus. Not sure of this phrasing.

First sentence is not changed - just here for context:

Maximum Bolus is the highest bolus amount Loop will recommend, or allow you to deliver, at one time to cover carbs or bring down high glucose.

The insulin Loop will automatically deliver is limited by an active insulin (IOB) amount that is twice your maximum bolus; manual bolus is not limited by this cap.

Reply:
or:

The insulin Loop will automatically deliver is limited by an active insulin cap equal to twice your maximum bolus. The active insulin from manual bolus is not limited by this cap.

Reply:
"This setting will also determine a safety limit for automatic dosing. Loop will limit automatic delivery to keep the amount of active insulin below twice your maximum bolus."

We already are talking about manual bolusing in the first sentence, and how that is capped. I think the wording above makes it a little clearer that this is a secondary effect that only affects automatic delivery.

Reply:
Agree. Looks good on the resultant screen too.